scripts/parse_anglerfish_results.py

```diff
@@ -13,127 +14,107 @@
 from epp_utils.udf_tools import fetch, put
 
 
-def get_anglerfish_output_file(lims: Lims, currentStep: Process, log: list):
+def find_latest_flowcell_run(currentStep: Process) -> str:
     flowcell_id: str = currentStep.udf["ONT flow cell ID"].upper().strip()
+    run_query = f"/srv/ngi-nas-ns/minion_data/qc/*{flowcell_id}*"
```

Comment:
It hurts my eyes a bit with a hardcoded path, but I do understand that it's easier to find and change it this way compared to as an input where the script is configured to run in the LIMS...

Reply:
There are a lot of hardcoded paths and configurations in this repo. I think we've previously discussed the benefits of turning it into a single tool with branching subcommands rather than a collection of scripts, which could include a centralized config same as we have for TACA.

Yes, getting the paths as CLI arguments in the EPP call in LIMS would make it more difficult to find and maintain in my opinion.

If you want I can declare the path variables at an earlier point of the script to make it a bit neater.

Reply:
yes I agree with your view. This will be fine for now.